README.md

```diff
@@ -51,13 +51,36 @@ const editor = new EditorJS({
 });
 ```
 
+### Add a custom shortcut to undo and redo
+
+**Note:** If you do not add any shortcut, the default shortcuts will be set up.
+
+If you want to add custom shortcuts, pass a config object with a shortcut key in the undo instance, the shortcuts must be called `undo` and `redo`.
+
+```javascript
+const config = {
+  shortcuts: {
+    undo: 'ctrlKey+x',
+    redo: 'ctrlKey+altKey+c'
```

Comment:
@MarioRodriguezS I noticed a common pattern in other plugins. When setting a shortcut they use `CMD` instead of `ctrlKey` or `metaKey`.

![image](https://user-images.githubusercontent.com/1649973/133513054-0808c651-29b9-4e48-ae89-36160331c084.png)

The plugin resolves the meta key internally depending on the operating system. 

May this work? what do you think?